packages/react-json-tree/src/index.tsx

```diff
@@ -47,133 +46,56 @@ const defaultLabelRenderer = ([label]: (string | number)[]) => (
 );
 const noCustomNode = () => false;
 
-function checkLegacyTheming(theme: Theme | undefined, props: Props) {
```

Comment:
@Methuselah96 As part of this update I removed support for the deprecated styling methods. Removing these is not strictly required to do in this change (just something that made the code easier to reason about). Since this could be seen as a breaking change I am happy to put this back if desired.